packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -988,23 +986,23 @@ class RenderListWheelViewport
   /// painting coordinates** system.
   @override
   void applyPaintTransform(RenderBox child, Matrix4 transform) {
-    final ListWheelParentData parentData = child?.parentData as ListWheelParentData;
+    final ListWheelParentData parentData = child.parentData as ListWheelParentData;
     transform.translate(0.0, _getUntransformedPaintingCoordinateY(parentData.offset.dy));
   }
 
   @override
-  Rect describeApproximatePaintClip(RenderObject child) {
+  Rect? describeApproximatePaintClip(RenderObject? child) {
     if (child != null && _shouldClipAtCurrentOffset()) {
```

Comment:
pretty sure child isn't allowed to be null here. maybe this line meant `this.child`?

Reply:
i would make the parameter non-nullable, and `assert(child == this.child); assert(child != null);` at the top of the method.

Reply:
Change to make param non-nullable